bazel/repository_locations.bzl

```
@@ -1,8 +1,8 @@
 REPOSITORY_LOCATIONS = dict(
-    # envoy 1.19.0
+    # envoy 1.19.0-patch2
```

let's update this comment to be clear which branch we depend on and why (https://github.com/solo-io/envoy-fork/commits/release/v1.19.0-patch)

```suggestion
    # we depend on envoy 1.19.0 with the cve patches applied on top (https://github.com/solo-io/envoy-fork/commits/release/v1.19.0-patch), since 1.19.1 would pick up changes that break our transformation filter and its unit tests
```

let's wait for ci to finish before updating though